packages/sanity/src/core/form/studio/FormBuilder.test.tsx

```diff
@@ -107,8 +109,9 @@ describe('FormBuilder', () => {
           schemaType: formState?.schemaType || schemaType,
           validation: EMPTY_ARRAY,
           value: formState?.value as FormDocumentValue,
+          useNewTreeDialog,
```

Comment:
Why are we passing this as a prop to the form builder?

Reply:
This needs to be updated after we made changes in formBuilder. We need it, afaik, because otherwise the formBuilder is rendering the wrong thing and the test is failing (since we added the conditional rendering in formBuilder.tsx)

Reply:
Updated this test (with the hook returning enabled `false`), I'm going to add another test for when it's true as well